Source/Core/Duality/Duality.csproj

```diff
@@ -117,6 +117,9 @@
     <Compile Include="Utility\Coroutines\CoroutineAction.cs" />
     <Compile Include="Utility\Coroutines\CoroutineManager.cs" />
     <Compile Include="Utility\Log\DebugLogOutput.cs" />
+    <Compile Include="Utility\Pooling\IPoolable.cs" />
```

Comment:
`Utility\Pooling` suggests a general, framework-wide pool implementation. Let's not do this for now - additional constraints would apply and suddenly we've got X use cases all affecting how pooling needs to be designed, and the thing gets a lot more complicated. There are several pooling implementations throughout the codebase, all of which we would suddenly need to consider. 

Finding the "one size fits all" implementation would be a big scope increase here. Let's keep it all Coroutine-specific, maybe even `internal`.

Reply:
> do you mean it should also be renamed to something like CoroutinePool?

Yep, so it will be obvious to users that this is a domain-specific implementation for internal usage. We can reconsider this at some later point, but one step at a time.